src/Npgsql/Common.cs

```diff
@@ -95,6 +40,21 @@ enum BackendMessageCode : byte
         RowDescription        = (byte)'T',
     }
 
+    static class FrontendMessageCode
+    {
+        internal const byte Describe = (byte)'D';
+        internal const byte Sync = (byte)'S';
+        internal const byte Execute = (byte)'E';
+        internal const byte Parse = (byte)'P';
+        internal const byte Bind = (byte)'B';
+        internal const byte Close = (byte)'C';
+        internal const byte Query = (byte)'Q';
+        internal const byte CopyDone = (byte)'c';
+        internal const byte CopyFail = (byte)'f';
+        internal const byte Terminate = (byte)'X';
+        internal const byte Password = (byte)'p';
+    }
```

Comment:
Obviously both are fine, but out of curiosity, why use `static`/`const` for the frontend messages, but an enum for the backend?

Reply:
I ended up with consts for the frontend codes simply because it removes the need to cast to byte every time we want to write the code. There isn't really such a concern with the backend codes... I also don't really care that much and can change if this bothers anyone.

Reply:
I have a personal preference for enums in general, but I was really just curious to hear your design reasoning for this case. 

Maybe @YohDeadfall has an opinion one way or another?

Reply:
FWIW I also use enums whenever possible, but in this case the *only* use of the values is to write them as bytes. In the BackendMessageCode case we switch on them, etc. etc.